docs/setup.md

```diff
@@ -55,20 +48,50 @@ After installation of the **Outbreak Response Framework** solution pack, run the
+
+    ![Select Integrations page](./res/config-wizard-02.png)
+
+5. **Configure Integrations**
+
+    - Select each integration's tab to configure the associated connector and data ingestion parameters. 
+
+        The page has separate tabs for each integration selected as a threat detection source:
+
+    ![Configure Integrations page](./res/config-wizard-03.gif)
+
+6. **Investigation Schedule**
+
+    ![Investigation Schedule page](./res/config-wizard-04.png)
+
+    - **Threat Hunt Window**: Specify the number of days as an interval within which outbreak should be considered for investigation.
+
+    - **Investigation Frequency**:  Select the frequency of the investigation to create a schedule. The created schedule can be found under *Automation* > *Schedules*. Once created, the schedule periodically runs the investigation on the reported outbreaks at the specified frequency.
+
+7. **Installation & Notification**
+
+    ![Installation and notification page](./res/config-wizard-05.png)
 
-    The hunt activities require searching for adversaries and their tactics, within an environment, against existing information available in the Threat Intel Platform's database (TIP). The Threat Detection Integration sources help run the threat hunt activities and are an important part of the *Outbreak Response Framework*.
+    - **Automatically install outbreak response solution packs**: Select this option to automatically install outbreak response solution packs once an outbreak is reported.
 
-    ![Define your Threat Detection Integrations](./res/config-wizard-02.png)
+    - **Auto Installation Criteria**: Select the severity of the outbreak to install the corresponding solution pack. You can select one or more severity from the following options:
+        - *Critical*
+        - *High*
+        - *Medium*
 
-5. Select Ticketing/ITSM Integration sources as part of your response or threat management strategy and click **Next**.
+    - **Outbreak Alert Update Notification**:  Specify email addresses authorized to receive outbreak updates. You can specify multiple email addresses separated by a comma.
 
-    Ticketing systems allow teams to organize, describe, and archive event investigations and incidents. For each hunt conducted, an integrated ticketing solution helps track and ease managing of alerts and incidents.
+8. **Summary**
 
-    ![Define your Ticketing/ITSM Integrations](./res/config-wizard-03.png)
+    Click the button **Ingest Now** to install the outbreak response solution packs of the severity selected on the previous screen.
 
-6. Click **Finish** to complete the configuration process.
-    ![All set](./res/config-wizard-04.png)
+    Click **Finish** to complete the configuration process.
+
+    ![All set](./res/config-wizard-06.png)
 
 ## Configure the Threat Detection Integration Parameters
 
```

Comment:
This needs to be revised entirely due to changes in the investigate playbook and the way we handle input parameters for all Threat Integrations.